lib/parallel_tests/cli.rb

```diff
@@ -192,7 +192,16 @@ def execute_shell_command_in_parallel(command, num_processes, options)
     def report_time_taken
       start = Time.now
       yield
-      puts "\nTook #{Time.now - start} seconds"
+      elapsed = Time.now - start
+      display_elapsed = display_duration(elapsed)
+      puts "\nTook #{elapsed} seconds" + (display_elapsed.include?(':') ? " (#{display_elapsed})" : "")
```

Comment:
true, it looks kind of silly to have `12 seconds (00:00:12)`

On Wed, Feb 25, 2015 at 4:46 PM, Ed Slocomb <notifications@github.com>
wrote:

> In lib/parallel_tests/cli.rb
> <https://github.com/grosser/parallel_tests/pull/366#discussion_r25395588>:
>
> > @@ -192,7 +192,16 @@ def execute_shell_command_in_parallel(command, num_processes, options)
> >      def report_time_taken
> >        start = Time.now
> >        yield
> > -      puts "\nTook #{Time.now - start} seconds"
> > +      elapsed = Time.now - start
> > +      display_elapsed = display_duration(elapsed)
> > +      puts "\nTook #{elapsed} seconds" + (display_elapsed.include?(':') ? " (#{display_elapsed})" : "")
>
> I wanted to leave the existing output undisturbed (there's got to be
> someone, somewhere parsing it with a script...) and didn't want to annoy
> users (who have somehow managed to keep their test suites under a minute)
> with redundant display.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/grosser/parallel_tests/pull/366/files#r25395588>.
>